packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -67,6 +67,9 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 - Added required \`variantId\` property to \`ProductApi\` interface.
 - Added optional \`taxLines\` property to \`ShippingLine\` interface.
 - Added optional \`onBlur\` handler to \`SearchBar\` component.
+- Added optional \`editable\` property to \`Cart\` interface.
+- Added \`useCartEditable\` hook to access the cart's editable state.
+- Added a new \`CartNotEditableError\` error class, thrown when attempting to modify the cart while it is not currently editable.
```

Comment:
I'm not sure if we want to include this here, it feels like an implementation detail more than a feature.